webboot/webboot.go

```diff
@@ -148,15 +132,25 @@ func main() {
 
 	//Processes the URL to receive an io.ReadCloser, which holds the content of the downloaded file
 	log.Println("Retrieving the file...")
-	read, err := linkOpen(URL)
+	iso, err := linkOpen(URL)
 	if err != nil {
 		log.Fatal(err)
 	}
 
```

Comment:
defer iso.Close()

Reply:
Okay, sounds good.  I did have a question when one should defer a Close() and when one should check an error, but after your explanation in one of the comments below, it makes more sense now :) thanks!